2_segmenting_data/README.md

```diff
@@ -1,4 +1,4 @@
-# Segment Schwann Cells from NF1 Data
+# 2. Segment NF1 Data
```

Comment:
Why remove Schwann cells? I thought this was important - maybe you talk about it later in this readme tho.

Reply:
The goal was to make it follow the other modules by adding the `2.`,  but I totally agree the "Schwann Cells" part is important and I don't mention it later.